### ISSUE TITLE
Fresh installation on Go 1.14 fails

### ISSUE THREAD

**Issue by Bjeaurn** [gopasspw/gopass#1252]
### Summary
Freshly installed Go 1.14, first command I tried to run after `go version` is the installation: `go get github.com/gopasspw/gopass` which resulted in the following:

```
┌ ❯ ~/Development ❯
└ ❯ × ❯ go version                       
go version go1.14 darwin/amd64
┌ ❯ ~/Development ❯
└ ❯ go get github.com/gopasspw/gopass

# github.com/gopasspw/gopass/pkg/completion/zsh
../go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:25:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.BoolFlag (Apply method has pointer receiver)
../go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:27:8: undefined: cli.BoolTFlag
../go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:29:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.Float64Flag (Apply method has pointer receiver)
../go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:31:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.GenericFlag (IsSet method has pointer receiver)
../go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:33:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.Int64Flag (Apply method has pointer receiver)
../go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:35:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.Int64SliceFlag (Apply method has pointer receiver)
../go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:37:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.IntFlag (Apply method has pointer receiver)
../go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:39:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.IntSliceFlag (Apply method has pointer receiver)
../go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:41:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.StringFlag (Apply method has pointer receiver)
../go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:43:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.StringSliceFlag (Apply method has pointer receiver)
../go/src/github.com/gopasspw/gopass/pkg/completion/zsh/completion.go:43:3: too many errors
# github.com/gopasspw/gopass/pkg/completion/fish
../go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:42:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.BoolFlag (Apply method has pointer receiver)
../go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:44:8: undefined: cli.BoolTFlag
../go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:46:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.Float64Flag (Apply method has pointer receiver)
../go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:48:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.GenericFlag (IsSet method has pointer receiver)
../go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:50:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.Int64Flag (Apply method has pointer receiver)
../go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:52:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.Int64SliceFlag (Apply method has pointer receiver)
../go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:54:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.IntFlag (Apply method has pointer receiver)
../go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:56:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.IntSliceFlag (Apply method has pointer receiver)
../go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:58:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.StringFlag (Apply method has pointer receiver)
../go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:60:3: impossible type switch case: f (type cli.Flag) cannot have dynamic type cli.StringSliceFlag (Apply method has pointer receiver)
../go/src/github.com/gopasspw/gopass/pkg/completion/fish/completion.go:60:3: too many errors
```

### Steps To Reproduce
See above, this was a fresh Go installation. Am using a custom prompt, will try again with a clean one.

### Expected behavior
The installation to run and not error out.

### Environment
- OS: MacOS Catalina 10.15.3
- OS version: 10.15.3
- gopass Version: Latest, installation didn't finish.
- Installation method: `go get github.com/gopasspw/gopass`

**Comment by creekorful** [comment]
I have had the same issue. Looks like gopass does not support Go 1.14 yet.

**Comment by acheron2302** [comment]
This git repository haven't been updated for decade there are many bugs. I make a pull request here to fix the problem but i don't think they will push or fix it soon: #1242 . For the temporary solution checkout my answer at this issue: #1248

**Comment by dominikschulz** [comment]
Should be fixed by #1273.
